### DISCUSSION TITLE
\usr\local\share\emacs\site-lisp not in emacs load-path

### DISCUSSION THREAD

**Discussion by ywwry66** [Homebrew/discussions#33] [Everyday usage] [answered]
I installed emacs with `brew cask install emacs`. If I run `emacs --no-init`, I got the following for `load-path`:
("/Library/Application Support/Emacs/27.1/site-lisp" "/Library/Application Support/Emacs/site-lisp" "/Applications/Emacs.app/Contents/Resources/site-lisp" "/Applications/Emacs.app/Contents/Resources/lisp" "/Applications/Emacs.app/Contents/Resources/lisp/vc" "/Applications/Emacs.app/Contents/Resources/lisp/url" "/Applications/Emacs.app/Contents/Resources/lisp/textmodes" "/Applications/Emacs.app/Contents/Resources/lisp/progmodes" "/Applications/Emacs.app/Contents/Resources/lisp/play" "/Applications/Emacs.app/Contents/Resources/lisp/org" "/Applications/Emacs.app/Contents/Resources/lisp/nxml" "/Applications/Emacs.app/Contents/Resources/lisp/net" "/Applications/Emacs.app/Contents/Resources/lisp/mh-e" "/Applications/Emacs.app/Contents/Resources/lisp/mail" "/Applications/Emacs.app/Contents/Resources/lisp/leim" "/Applications/Emacs.app/Contents/Resources/lisp/language" "/Applications/Emacs.app/Contents/Resources/lisp/international" "/Applications/Emacs.app/Contents/Resources/lisp/image" "/Applications/Emacs.app/Contents/Resources/lisp/gnus" "/Applications/Emacs.app/Contents/Resources/lisp/eshell" "/Applications/Emacs.app/Contents/Resources/lisp/erc" "/Applications/Emacs.app/Contents/Resources/lisp/emulation" "/Applications/Emacs.app/Contents/Resources/lisp/emacs-lisp" "/Applications/Emacs.app/Contents/Resources/lisp/cedet" "/Applications/Emacs.app/Contents/Resources/lisp/calendar" "/Applications/Emacs.app/Contents/Resources/lisp/calc" "/Applications/Emacs.app/Contents/Resources/lisp/obsolete")

While in emacs.rb, https://github.com/Homebrew/homebrew-core/blob/ddd4f0b1a030b98e065874e137e3d2fd604b3f41/Formula/emacs.rb#L41 seems to fail its purpose.

**Comment by SMillerDev** [comment, marked as answer]
`brew cask install emacs` is not using the code in the link you posted. It would use https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/emacs.rb.

The link you posted would be used by `brew install emacs`

**Reply by ywwry66** [reply]
Thanks. You are absolutely right. I don't know what I was thinking.
